tools/spice/spice_examples.py

```diff
@@ -0,0 +1,132 @@
+"""Annotated Events: project events in instrument frame to reference systems.
+
+Reference: https://spiceypy.readthedocs.io/en/main/documentation.html.
+"""
+
+import logging
+
+import spiceypy as spice
+
+from tools.spice.spice_utils import list_all_constants
+
+# Logger setup
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def get_attitude_timerange(ck_kernel, id):
+    """Get attitude timerange using the ck kernel.
+
+    Parameters
+    ----------
+    ck_kernel: str
+        Directory of the ck kernel.
+    id: int
+        ID of the instrument or spacecraft body.
+
+    Returns
+    -------
+    start: float
+        Start time in ET.
+    end: float
+        End time in ET.
+    """
+    # Get the first CK path
+    ck_path = ck_kernel[0]
+
+    # Get the IDs from the CK file
+    ck_id = spice.ckobj(ck_kernel[0])
+
+    # Check if the specified ID is in the CK file
+    if int(id) not in ck_id:
+        raise ValueError(f"ID {id} not found in CK file")
+
+    # Get the coverage window
+    cover = spice.ckcov(ck_path, int(id), True, "SEGMENT", 0, "TDB")
```

Comment:
Can you add links like this above these functions for reference or for anyone who doesn't know about this yet?
https://spiceypy.readthedocs.io/en/main/documentation.html#spiceypy.spiceypy.ckcov

I know it may be duplicate effort but knowing what those parameters are very helpful.